gen-resourcesdocs/config/v1.21/toc.yaml

```diff
@@ -0,0 +1,274 @@
+# Copyright 2016 The Kubernetes Authors.
+# Copyright 2020 Philippe Martin
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+parts:
+- name: Workloads Resources
+  chapters:
+  - name: Pod
+    group: ""
+    version: v1
+    otherDefinitions:
+    - PodSpec
+    - Container
+    - Handler
+    - NodeAffinity
+    - PodAffinity
+    - PodAntiAffinity
+    - Probe
+    - PodStatus
+    - PodList
+  - name: EphemeralContainers
+    group: ""
+    version: v1
+    otherDefinitions:
+    - EphemeralContainer
+  - name: PodTemplate
+    group: ""
+    version: v1
+  - name: ReplicationController
+    group: ""
+    version: v1
+  - name: ReplicaSet
+    group: apps
+    version: v1
+  - name: Deployment
+    group: apps
+    version: v1
+  - name: StatefulSet
+    group: apps
+    version: v1
+  - name: ControllerRevision
+    group: apps
+    version: v1
+  - name: DaemonSet
+    group: apps
+    version: v1
+  - name: Job
+    group: batch
+    version: v1
```

Comment:
In the left hand navigation tree, is each list item created by concatenating the version and resource name strings together, excluding resources with version `v1`? Resources with `v1` only use the resource name. For example, in the left hand navigation tree:
Pod
Job
CronJobv1beta1
CronJobv2alpha1

Wondering if all the versions should be labelled.